### PR TITLE
Adding pro-tip when working with clean buffers

### PR DESCRIPTION
**Pull request by mike3run**
Just some cool info i stumbled upon that might be useful to know in the readme 😉

### CODE REVIEW

**Comment by mike3run** [comment]
The text I added to the readme is this below: ⬇️ 

Note: This plugin won't change the syntax if you start with an empty buffer and then add styled-components code, you will need to close the buffer and re-open it so that the styles apply.

A quick way to do that is with this command: :edit! which will do exactly that. You can map that to a shortcut if you wish.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ This plugin checks, whether the current buffer contains an `import` or a `requir
 
 Note however, that this plugin does not provide its own CSS rules. You should have an up to date vim runtime (for CSS2 rules) and - as long as CSS3 rules are not included - an additional plugin for CSS3 rules. (I recommend [hail2u/vim-css3-syntax](https://github.com/hail2u/vim-css3-syntax) but feel free to use whatever you like.)
 
+**Note:** This plugin won't change the syntax if you start with an empty buffer and then add styled-components code, you will need to close the buffer and re-open it so that the styles apply.
```
Comment (fleischie) [inline README.md]: Albeit valuable information I think there should be some changes done:
- Don't make this Note bold. This unevenly distributes the attention towards this note (there is another equally important note above).
- This plugin won't change the syntax at all, it only highlights the syntax. I'd like it to say this as well.
- Make this paragraph two sentences. The first until the first comma and the subsequent part as the second.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ This plugin checks, whether the current buffer contains an `import` or a `requir
...
+
+A quick way to do that is with this command: `:edit!` which will do exactly that. You can map that to a shortcut if you wish.
```
Comment (fleischie) [inline README.md]: Also:
- Maybe merge these two paragraphs, as they relate to the same issue (and not two separate ones, as it reads now).
- You can also write `:filetype detect`.
- This plugin only affects files, that contain certain content (`import <something> from `styled-components` for example). So re-opening/re-evaluating the file is no guarantee to highlight the styled-components syntax.

Comment (mike3run) [reply README.md]: @fleischie so merging paragraphs like you say here or separate paragraphs as said before?

Also that filetype detect command was super awesome haha thanks for that

**Comment by mike3run** [comment]
I no longer use vim I have fled to Atom so this PR is bound to stay in limbo forever.

Thanks for the plugin  tho

**Comment by fleischie** [comment]
@mike3run sorry to hear that. I hope you enjoy Atom and maybe give some vim-mode plugins a chance. :smile:

:wave:

**Comment by mike3run** [comment]
Oh you know it haha full vim mode in my atom, im actually enjoying it a lot

**Pull request closed without merging**